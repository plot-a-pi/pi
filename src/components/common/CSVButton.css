.CSVButton {
  float: right;
  position: relative;
  bottom: 3vw;
  right: 0.5vw;
  width: 8vw;
  background:rgb(179, 181, 196);
  font-size: 1.5vw;
  z-index: 1;
  text-align: center;
  border-radius: .5rem;
  padding: .5vw;
  color: #212E59;
}

.CSVButton:hover {
  float: right;
  position: relative;
  bottom: 3vw;
  right: 0.5vw;
  width: 8vw;
  background: rgb(255, 255, 255);
  font-size: 1.5vw;
  text-align: center;
  border-radius: .5rem;
  padding: .5vw;
  color: #212E59;
}