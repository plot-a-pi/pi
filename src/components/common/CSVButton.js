import React from 'react';
import CSV from './CSV';

const CSVButton = ({ children }) => {

  return (
    <>
      <button> Download Icon
        {children}
      </button>
    </>
  );

};

export default CSVButton;
