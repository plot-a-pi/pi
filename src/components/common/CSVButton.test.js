import React from 'react';
import { shallow } from 'enzyme';
import CSVButton from './CSVButton';

describe('CSVButton component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<CSVButton data={[{ x: 1, y: 2 }]} header1={'test header 1'} header2={'test header 2'}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
