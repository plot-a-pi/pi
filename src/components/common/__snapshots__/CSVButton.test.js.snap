// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSVButton component matches a snapshot 1`] = `
<div
  className="CSVButton"
  onClick={[Function]}
>
  Raw Data
</div>
`;
